include/exiv2/heifimage.hpp

```diff
@@ -0,0 +1,72 @@
+#ifndef HEIFIMAGE_HPP
+#define HEIFIMAGE_HPP
+
+#include "basicio.hpp"
+#include "exif.hpp"
+#include "image.hpp"
+#include "types.hpp"
+
+#include <string>
+
+#include <libheif/heif.h>
```

Comment:
Are you actually using the library ? I do not see where is used in the header.

Reply:
Yes, finally I've added HEIF library calls to obtain width and height of the image.

Reply:
But it is not used in the header right ? only in the .cpp. If that's the case, we should move this inclusion to the .cpp file.